<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Bogdan Marinov</author>
 <class>LocationListEditor</class>
 <widget class="QMainWindow" name="LocationListEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView">
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frameFilter">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="lineEditFilter">
         <property name="placeholderText">
          <string>Search/filter...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBoxFilteredColumn"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLocations">
    <property name="title">
     <string>&amp;Locations</string>
    </property>
    <addaction name="actionOpenProject"/>
    <addaction name="actionOpenUser"/>
    <addaction name="actionOpenFile"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionBinary"/>
    <addaction name="actionNativeEolChars"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionNewLocation"/>
    <addaction name="action_Clone_selected"/>
    <addaction name="separator"/>
    <addaction name="actionDelete"/>
   </widget>
   <addaction name="menuLocations"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpenProject">
   <property name="text">
    <string>Open &amp;current project locations</string>
   </property>
  </action>
  <action name="actionOpenUser">
   <property name="text">
    <string>Open current &amp;user locations</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>&amp;Open locations file...</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="toolTip">
    <string>Save the current location list</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNewLocation">
   <property name="text">
    <string>&amp;New location</string>
   </property>
   <property name="iconText">
    <string>Insert a new location</string>
   </property>
   <property name="toolTip">
    <string>Insert a new location</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cu&amp;t</string>
   </property>
   <property name="toolTip">
    <string>Cut </string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>&amp;Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>&amp;Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About...</string>
   </property>
  </action>
  <action name="actionBinary">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Also save as a &amp;binary file</string>
   </property>
  </action>
  <action name="actionShowName">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Name</string>
   </property>
   <property name="toolTip">
    <string>Show the Name column</string>
   </property>
  </action>
  <action name="actionShowRegion">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Region</string>
   </property>
   <property name="toolTip">
    <string>Toggle the Region column</string>
   </property>
  </action>
  <action name="actionShowCountry">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Country</string>
   </property>
   <property name="toolTip">
    <string>Toggle the Country column</string>
   </property>
  </action>
  <action name="actionShowType">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Type</string>
   </property>
   <property name="toolTip">
    <string>Toggle the Type column</string>
   </property>
  </action>
  <action name="actionShowPopulation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Population</string>
   </property>
   <property name="toolTip">
    <string>Show the Population column</string>
   </property>
  </action>
  <action name="action_Clone_selected">
   <property name="text">
    <string>&amp;Clone selected</string>
   </property>
  </action>
  <action name="actionNativeEolChars">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use native line endings</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
